paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -188,6 +205,7 @@ def paasta_start_or_stop(args, desired_state):
         # actually within this cluster.
         if instances is None:
             cluster_instances = list_all_instances_for_service(service, clusters=[cluster], soa_dir=soa_dir)
+            print("no instances specified; restarting all instances for service")
```

Comment:
Can you update the help text to make this obvious that it is the behavior? (which will translate to the man page)